files/zh-tw/web/http/status/304/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: 304 Not Modified
+slug: Web/HTTP/Status/304
+l10n:
+  sourceCommit: fb8c438501d119fb897a9667ee85c1486f2c03d6
+---
+
+{{HTTPSidebar}}
+
+HTTP **`304 Not Modified`** 重新導向回應碼表示無需重新傳送所請求的資源。
+當請求是帶有 {{HTTPHeader("If-None-Match")}} 或 {{HTTPHeader("If-Modified-Since")}} 標頭的條件式 {{HTTPMethod("GET")}} 或 {{HTTPMethod("HEAD")}} 請求，並且該條件求值為 false 時，會發送此回應碼。如果條件求值為 true，則這是對一個已緩存的資源的隱式重定向，而該資源的請求如果沒有條件，則會導致一個 {{HTTPStatus("200")}} `OK` 的回應。
```

Comment:
不必分行

```suggestion
HTTP **`304 Not Modified`** 重新導向回應碼表示無需重新傳送所請求的資源。當請求是帶有 {{HTTPHeader("If-None-Match")}} 或 {{HTTPHeader("If-Modified-Since")}} 標頭的條件式 {{HTTPMethod("GET")}} 或 {{HTTPMethod("HEAD")}} 請求，並且該條件求值為 false 時，會發送此回應碼。如果條件求值為 true，則這是對一個已緩存的資源的隱式重定向，而該資源的請求如果沒有條件，則會導致一個 {{HTTPStatus("200")}} `OK` 的回應。
```